AUTHORS

```diff
@@ -110,25 +131,29 @@ MCApollo             <34170230+MCApollo@users.noreply.github.com>
 Marc Abramowitz      <marc@marc-abramowitz.com>
 Marc Bruggmann       <marcbr@spotify.com>
 Marcin Kłopotek      <marcin.klopotek@gmail.com>
+Marcin Serwin        <marcin@serwin.dev>
 Mark Feit            <mfeit@internet2.edu>
 Mark McKinstry       <mmckinst@umich.edu>
 Markus Lanthaler     <mark_lanthaler@gmx.net>
 Matt Clegg           <m@cle.gg>
+Matt Meyers          <matt@mattmeyers.dev>
 Matti Åstrand        <mattiastr@gmail.com>
 Mattias Hansson      <hansson.mattias@gmail.com>
 Maxime Biais         <maxime.biais@gmail.com>
 Maximilian Roos      <5635139+max-sixty@users.noreply.github.com>
 Michael Daines       <michael@mdaines.com>
+Michael Färber       <01mf02@gmail.com>
 Mike Daines          <mdaines@users.noreply.github.com>
 Mike Fletcher        <john.m.fletcher@bankofamerica.com>
 Mike McCabe          <mccabe@archive.org>
+Nathan Baulch        <nathan.baulch@gmail.com>
 Naïm Favier          <n@monade.li>
 Nicolas Pouillard    <nicolas.pouillard@gmail.com>
+Nicole Wren          <nicole@wren.systems>
 Paul Chvostek        <paul+gitkraken@it.ca>
 Paul Wise            <pabs3@bonedaddy.net>
 Peter van Dijk       <peter@7bits.nl>
 Philipp Hagemeister  <phihag@phihag.de>
-Rémy Léone           <remy.leone@gmail.com>
```

Comment:
Sorting changed from Unicode-aware to ASCII, so this contributor is now sorted after Rz. Should this be fixed?

Reply:
I'm not sure how to get the original order (Unicode-aware and case sensitive).  This time I used `git log --pretty=format:"%<(20)%an <%ae>" | awk -F '<' '!x[$1]++' | LC_COLLATE=C sort | gsed -nE '/Stephen Dolan|dtolnay|emanuele|leonid|wadman|muhten|cryptonector|owenou|wlangfor|itchyny|pkoppstein/!H; //p; $x; $p'`.

Reply:
It could be a Unicode composition/decomposition difference. In any case, it doesn't matter.